iib/workers/tasks/legacy.py

```diff
@@ -36,8 +36,11 @@ def export_legacy_packages(packages, request_id, rebuilt_index_image, cnr_token,
             _verify_package_info(package_dir, rebuilt_index_image)
             _zip_package(package_dir)
             _push_package_manifest(package_dir, cnr_token, organization)
-
-    set_request_state(request_id, 'in_progress', 'Back ported packages successfully pushed to OMPS')
```

Comment:
Please revert back your change in this method so that the method continues using `set_request_state`. It's a wrapper around `update_request` so that you don't have to specify the payload like you did below.

Reply:
yeah, I used update_request since earlier I added something in payload, but with feedback I realized that wasn't needed, now I can use the set_request_state() directly, changing in next patch